source/loader/ze_loader.cpp

```diff
@@ -200,46 +200,6 @@ namespace loader
             return ZE_RESULT_ERROR_UNINITIALIZED;
         }
 
-        if(nullptr != validationLayer) {
```

Comment:
Are these sections not needed at all anymore?

Reply:
These sections were being extra careful by checking that not only did the call to zeInit() pass going to the driver, but thru the validation and tracing layers if they exist.

This check should not be required. The only time this check caught anything is if one had an invalid set of layers installed, but then the error would occur at zeInit() time frame instead of during the init phase.

The addition of the handling for at least one driver success in zeDriverGet removes the requirement for these checks.

Reply:
sounds good